### PR TITLE
misc: rename license header to use Lighthouse Authors

### PR DESCRIPTION
**Pull request by patrickhulce**
**Summary**
Ready to go pending approval in upcoming Lighthouse Authors meeting :)

https://opensource.google/docs/releasing/contributions/#copyright states that projects with high volume of external contributions may use "The *insert project name* Authors" instead of "Google Inc." in the copyright header and supply an `AUTHORS` file at the root of the repo where contributors may optionally add their name if they wish to be recognized. We already have such a file and contributors are free to open PRs to add themselves if they so desire.

Created with 
```
find . -type f \( -name "*.js" -o -name "*.css" -o -name "*.html" -o -name "*.d.ts" -o -name "*.hbs" -o -name "*.rb" -o -name "*.sh" -o -name "*.json" \) \! -path "./node_modules/*" \! -path "./.git/*" | xargs gsed -i'' 's/Google Inc. All Rights Reserved./The Lighthouse Authors. All Rights Reserved./g'
git checkout master lighthouse-core/test/results/artifacts/artifacts.json
```

**Related Issues/PRs**
https://github.com/GoogleChrome/lighthouse/pull/10460#discussion_r392512238

### CODE REVIEW

**Comment by connorjclark** [comment]
heh, smoke test failed (bytes)

File: lighthouse-core/scripts/legacy-javascript/summary-sizes.txt [hunk 1]
```diff
@@ -239,6 +239,6 @@ variants/core-js-3-preset-env-esmodules
 variants/only-plugin
 2770 total
 1152 variants/only-plugin/-babel-plugin-transform-spread/main.bundle.min.js
+ 827 variants/only-plugin/-babel-plugin-transform-regenerator/main.bundle.min.js
```
[the remark this answers is not in the document]
Comment (patrickhulce) [reply lighthouse-core/scripts/legacy-javascript/summary-sizes.txt]: no this was completely unexpected, but given that this test is only run when certain files get touched (and this PR touched all of them), i'm assuming it shifted ever so slightly due to a yarn update at some point.

if we care about being more rigorous here we can try and trace down the bisect and add those files to our set of "changed files" to test on, but personally I don't think it's critical enough.

@connorjclark do you disagree?

[the remark this answers is not in the document]
Comment (brendankenny) [reply lighthouse-core/scripts/legacy-javascript/summary-sizes.txt]: ah, got it, I assumed this was also something about changing byte length due to the license  change, but that's not supported by the file name I guess :)

I'm not so worried about rigor for this PR, but stability is something we should think about for future PRs from folks who didn't write or review the generator for this test file so won't know what to make of a change in this file :)

[the remark this answers is not in the document]
Comment (connorjclark) [reply lighthouse-core/scripts/legacy-javascript/summary-sizes.txt]: maybe we need a lockfile for scripts/legacy-javascript/package.json

**Pull request merged**